src/OpenLoco/Map/RoadTile.cpp

```diff
@@ -0,0 +1,81 @@
+#include "../CompanyManager.h"
+#include "../GameCommands/GameCommands.h"
+#include "../Interop/Interop.hpp"
+#include "../Objects/RoadObject.h"
+#include "Tile.h"
+#include "TileManager.h"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::Map
+{
+    static loco_global<uint32_t, 0x00525FBC> _525FBC;
+
+    // 0x00477FC2
+    bool RoadElement::update(const Map::Pos2& loc)
+    {
+        if (owner() == CompanyId::neutral || isPlayerCompany(owner()))
+            return true;
+
+        if (!(_525FBC & (1 << roadObjectId())))
+            return true;
+
+        if (sequenceIndex())
+            return true;
+
+        if (mods())
```

Comment:
after looking at this again i think its also check something else. Only bits 6 and 7 are mods for road elements (its 4,5,6,7 for track elements). In the assembly this was also checking bits 4 and 5 for non zero. I haven't come across anything so far that uses bits 4 and 5 of road elements though so its hard to know if this is needed.

Reply:
bit 5 of road element is used for level crossing.

Reply:
Shall we just amend add `isLevelCrossing` to check for bit 5 and a placeholder `isBit6()`, then?

Reply:
Yeah i think so. Bit4 placeholder. It's not used in the drawing routine but that doesn't mean it's not used